lib/internal/safe_globals.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const copyProps = (unsafe, safe) => {
+  for (const key of [...Object.getOwnPropertyNames(unsafe),
+                     ...Object.getOwnPropertySymbols(unsafe)
+  ]) {
+    if (!Object.getOwnPropertyDescriptor(safe, key)) {
+      Object.defineProperty(
+        safe,
+        key,
+        Object.getOwnPropertyDescriptor(unsafe, key));
+    }
+  }
+};
+const makeSafe = (unsafe, safe) => {
```

Comment:
These should be `function () {}`s?

(Please don't export arrow functions unless absolutely necessary.)